apps/modernization-ui/src/setupProxy.js

```diff
@@ -11,8 +11,21 @@ const LOGIN = function (pathname, req) {
 };
 
 module.exports = function (app) {
+    // Modernization API
     app.use(
-        createProxyMiddleware([NBS_API, GRAPHQL, ENCRYPTION, PAGEBUILDER_API], { target: 'http://localhost:8080/' })
+        createProxyMiddleware([NBS_API, GRAPHQL, ENCRYPTION, PAGEBUILDER_API], {
+            target: process.env.npm_config_backend,
```

Comment:
issue:  This prevents the `npm run start` from running if the environment variable is not set.  Can you include `http://localhost:8080/` as a default?

Reply:
Oh no my intention is not to commit this. Reverting it back.